server/galaxyls/tests/files/simple_conditional_02.xml

```diff
@@ -0,0 +1,15 @@
+<tool id="simple_conditional_02" name="Simple conditional input with boolean">
+    <inputs>
+        <conditional name="bool_conditional" >
+            <param name="option" type="boolean" truevalue="yes" falsevalue="no" checked="false" />
```

Comment:
I'm on the fence here ... in theory, this is valid and I see tools doing this. But the IUC discourages it - and recommends using a select even if its just two elements.

But I guess its fine to leave it in.

Reply:
Ok, but I am happy to remove it if it is not considered a good practice. Less code, fewer bugs :)

Reply:
its valid code, so not sure ... what about adding a warning into the generated code ... using booleans as a conditional selector is not encouraged, please use a select parameter.

Reply:
Good idea! that can be easily done.